frontend/src/utils/formatting.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * This is thin wrapper on top of the formatter that's passed to
+ * reat-timeago. The desired date is a format that's something like
```

Comment:
```suggestion
 * react-timeago. The desired date is a format that's something like
```

Reply:
Good catch @ddhanesha! Here's an example of the GitHub "suggestion" feature for spelling suggestions if you hadn't seen it before. It's super useful IMO.